test/integration/features/che.feature

```diff
@@ -1,53 +1,53 @@
-@che
+@che @addon-che @addon
```

Comment:
Any specific reason to keep both tag ```@addon-che``` and  ```@addon```

Reply:
for later integration where we also use addon-eap-cd, not only eap-cd

Reply:
ahh..sorry i suppose to say ```@che``` and ```@addon-che```

Reply:
Yeah `@che` and `@addon-che` seems redundant.